packages/realm_dart/.gitignore

```diff
@@ -1,4 +1,3 @@
 /binary/
 /build-native/
 /mongodb-realm/
-*.realm.*
```

Comment:
🤦  .. this was added to exclude the actual `realm` database files, not the generated `.realm.dart` files.. things has been a bit hectic lately